src/main/java/org/jitsi/videobridge/health/Health.java

```diff
@@ -238,6 +239,9 @@ public Health(Videobridge videobridge)
     {
         super(videobridge, HealthConfig.getInterval(), true);
 
+        System.out.println(NewHealthConfig.getTimeout());
```

Comment:
Is this intentional?

Reply:
This is more of a draft (esp the jvb one), so probably lots of dump little stuff like this in here